tests/unit/models/category-test.js

```diff
@@ -10,3 +11,17 @@ test('it exists', function(assert) {
   // let store = this.store();
   assert.ok(!!model);
 });
+
+test('it should have all its attributes', function(assert) {
+  assert.expect(3);
+
+  const category = this.subject();
+  const attributes = Object.keys(category.toJSON());
+
+  assert.ok(attributes.includes('description'), 'should have description attribute');
+  assert.ok(attributes.includes('name'), 'should have name attribute');
+  assert.ok(attributes.includes('slug'), 'should have slug attribute');
```

Comment:
We should refactor these into an `assert.hasAttribute(attributes, 'attribute'`) which itself just runs `assert.ok()` with the description we use there.

Let's make an issue to circle back there. Will DRY these up slightly.